algonaut_core/src/lib.rs

```diff
@@ -401,4 +359,38 @@ mod tests {
 
         assert!(invalid_csum.parse::<Address>().is_err());
     }
+
+    /// PartialEq is implemented correctly (signatures are equal)
+    #[test]
+    fn signatures_equal() {
```

Comment:
Possible shorter version: `assert_eq!(Signature([0; 64]), Signature([0; 64]))`. But explicit (and not repeated) is better.